include/dsn/tool/latency_tracer.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include <string>
+#include <map>
+#include <dsn/service_api_c.h>
+#include <dsn/dist/fmt_logging.h>
+
+namespace dsn {
+namespace tool {
+
+struct latency_tracer
```

Comment:
Will a single tracer trace in multiple threads? If not, a thread local would be better.

Reply:
`thread local` must be used as `global`, in `class` need declare `static`, but the variable is `member variable` of the class